packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3216,10 +3216,19 @@ export class ContainerRuntime
 			"Fluid.ContainerRuntime.Test.SummarizationRecoveryMethod",
 		);
 		if (recoveryMethod === "restart") {
-			this._summarizer?.stop("latestSummaryStateStale");
-			// TODO: stop execution, but don't cause a cascading series of errors.
 			const error = new GenericError("Restarting summarizer instead of refreshing");
-			this.closeFn(error);
+			this.mc.logger.sendTelemetryEvent(
+				{
+					...event,
+					eventName: "RestartInsteadOfRefreshFromServerFetch",
+					originatingEventName: event.eventName,
+					message: "Stopping fetch from storage",
+					versionId: versionId != null ? versionId : undefined,
```

Comment:
Is `null` not allowed in telemetry props?

Reply:
Yeah, it wouldn't allow it